requirements.txt

```diff
@@ -2,6 +2,6 @@ RadeonOpenCompute/rocm-cmake@04f694df2a8dc9d7e35fa4dee4ba5fa407ec04f8 --build
 sqlite3@3.17 -DCMAKE_POSITION_INDEPENDENT_CODE=On
 boost@1.79 -DCMAKE_POSITION_INDEPENDENT_CODE=On --build
 half,https://github.com/pfultz2/half/archive/1.12.0.tar.gz -X header -H sha256:0a08660b68abb176ebc2a0cdf8de46e3182a7f46c66443bb80dbfaaec98cf969 --build
-ROCmSoftwarePlatform/llvm-project-mlir@rocm-5.3.0 -DBUILD_FAT_LIBMLIRMIOPEN=1
+ROCmSoftwarePlatform/rocMLIR@release/rocm-5.4  -DBUILD_FAT_LIBROCKCOMPILER=1
```

Comment:
@jerryyin and @zhanglx13 can you point to an actual commit ID instead of a branch name? Unless `release/rocm-5.4` is a release tag already.

Reply:
11f2e26f193d398cc67f00d9b8e5a58504a14092, [here](https://github.com/ROCmSoftwarePlatform/rocMLIR/commit/11f2e26f193d398cc67f00d9b8e5a58504a14092) is the last commit of ROCm 5.4 branch. You are right that we haven't tagged rocm-5.4.0 yet.

Reply:
@jerryyin MIOpen CI docker has been updated; could you update this line with commit hash? I would expect the next round of CI to pass :)

Reply:
@junliume Which docker image is the latest one? `rocm/miopen:ci`?

Reply:
@zhanglx13 yes `rocm/miopen:ci` has the correct CK so this PR should go through; however, we need this PR in so that we can create a new docker image which installed rocMLIR@11f2e26f193d398cc67f00d9b8e5a58504a14092. ETA this Friday.